app/controllers/leaderboard_controller.rb

```diff
@@ -1,6 +1,7 @@
 class LeaderboardController < ApplicationController
   def users
-    @leaderboards = UserLeaderboard.includes(:user).week(start_date, end_date)
+    @leaderboards = UserLeaderboard.includes(:user).week(start_date, end_date).
+      with_logged_minutes
```

Comment:
@schmierkov We usually use Option A for multi-line method chaining: https://github.com/bbatsov/ruby-style-guide#consistent-multi-line-chains